src/hearings/containers/hearing-actuals/hearing-actuals-add-edit-summary/hearing-actuals-add-edit-summary.component.ts

```diff
@@ -25,13 +25,13 @@ export class HearingActualsAddEditSummaryComponent extends HearingActualsSummary
     this.partyChannels = [...this.route.snapshot.data.partyChannels, ...this.route.snapshot.data.partySubChannels];
   }
 
-  /**
-   * TODO: Navigate to check your answers page if not cancelled and valid
-   * Determines whether submit hearing details on
-   */
-  public async onSubmitHearingDetails(): Promise<void> {
+  public onSubmitHearingDetails(): void {
     if (this.hearingResult === HearingResult.CANCELLED || this.isValid()) {
-      await this.router.navigate(['/', 'hearings', 'actuals', this.hearingRequestID, 'hearing-actual-edit-summary']);
+      // Had to add then() due to sonarcloud failure
+      this.router.navigate(['/', 'hearings', 'actuals', this.hearingRequestID, 'hearing-actual-edit-summary']).then(
```

Comment:
@Daniel-Lam - I remember you provided me this solution when Sonarcloud was not happy without `then`. This is in place only due to the Sonarcloud failure.

Reply:
If I take `then` out then Sonarcloud will again fail for this PR, it is inconsistent.

Reply:
OK - makes sense, @johnbenjamin-hmcts... would it work with an empty function instead of `console.log` and `console.error`, seeing as we don't really need these?

Reply:
@Daniel-Lam - it worked without `then` this time :)